tests/Api/Responses/admin/taxon/get_taxon_response.json

```
@@ -13,7 +13,7 @@
     ],
     "images": [
         {
-            "@id": "\/api\/v2\/admin\/taxon-images\/@integer@",
+            "@id": "\/api\/v2\/admin\/taxons\/@string@\/images\/@integer@",
```

You can use here a specific taxon code, the same in below responses
```suggestion
            "@id": "\/api\/v2\/admin\/taxons\/CATEGORY\/images\/@integer@",
```